### DISCUSSION TITLE
ALARM with second

### DISCUSSION THREAD

**Discussion by navivfr** [Makuna/Rtc#201] [Q&A] [answered]
Hello,

We use  PCF8563.
For our project we need to set up the ALARM with second.
Will be  possible in further update ?

PS : in documentation, the register address is 0x0B.

BR

**Comment by Makuna** [comment, marked as answer]
The PCF8563 does not support an alarm with the accuracy of a second.  Only the top of the minute is the finest quantum. 

Register 0x0B is Day_Alarm.  The day of the month.

You could implement it by cascading an Alarm and Timer.  When the alarm triggers set a Timer with the seconds and when the timer triggers then do your final work.

**Reply by navivfr** [reply]
The point is I have 5 IOT that send data over radio every hours (then the CPU goes off until the clock send an alarm).
This transaction take more or less 2 seconds.
In order to avoid "coiled" data, I need, each hour, one to connect at xxh00:00s, the next one at xxh00:05s, xxh00:10s..
With timer can appear a jitter... since it start at the moment when is setup.
If the alarm is set at 15h00 and the clock is 14h00:10s... the alarm will occur at 15h00:00 or at 15h00:10s ?

**Reply by navivfr** [reply]
all system (CPU and probe) goes off... in order to preserve the battery.
When OFF it consume like 50uA...

**Reply by navivfr** [reply]
I found a really nasty way to do it:
Since at each connection I resync the clock, I just need to remove  the seconds I need..
Since the time is set with seconds (SetDateTime)
tadaaa

**Reply by Makuna** [reply]
@navivfr Thats a novel solution.  Great to hear you found a workable solution.
